guides/source/upgrading_ruby_on_rails.md

```diff
@@ -81,6 +81,24 @@ Upgrading from Rails 7.0 to Rails 7.1
 Upgrading from Rails 6.1 to Rails 7.0
 -------------------------------------
 
+### `ActionView::Helpers::UrlHelper#button_to` changed behavior
+
+If your application uses `button_to` helper with an Array that contains an Active Record object as the last element. Starting from Rails 7.0 `button_to` renders a `form` tag with `patch` HTTP verb for a persisted object.
```

Comment:
Hmmm, it looks like this happens even if you don't pass an array, according to the example in https://github.com/rails/rails/pull/43413

Reply:
```suggestion
Starting from Rails 7.0 `button_to` renders a `form` tag with `patch` HTTP verb if any of its arguments are a persisted ActiveRecord object.
```

Reply:
Changed